src/btree/bt_sync_obsolete.c

```diff
@@ -150,7 +150,7 @@ __sync_obsolete_inmem_evict_or_mark_dirty(WT_SESSION_IMPL *session, WT_REF *ref)
           session, newest_ta.newest_stop_txn, newest_ta.newest_stop_durable_ts);
 
     if (obsolete) {
-        __wt_verbose(session, WT_VERB_CHECKPOINT_CLEANUP,
```

Comment:
All these messages are per page, they should be debug2.

Reply:
@kommiharibabu I have updated all the verbose messages in `bt_sync_obsolete.c` to `__wt_verbose_debug2`.